resources/measures/upgrade_df_lighting_control/measure.rb

```diff
@@ -0,0 +1,425 @@
+    #   runner.registerAsNotApplicable('Measure is not applied based on user input.')
+    #   return true
+    # end
+
+    ############################################
+    # Load prediction
+    ############################################
+    puts("### Reading weather file...")
+    oat = read_epw(model)
+    puts("--- oat.size = #{oat.size}")
+    if load_prediction_method == 'full baseline'
+      puts("### ============================================================")
+      puts("### Running full baseline for load prediction...")
+      annual_load = load_prediction_from_full_run(model, num_timesteps_in_hr=num_timesteps_in_hr)
+      # puts("--- annual_load = #{annual_load}")
+      puts("--- annual_load.size = #{annual_load.size}")
+    elsif load_prediction_method.include?('bin sample')
+      puts("### ============================================================")
+      if load_prediction_method == 'bin sample'
+        puts("### Creating bins...")
+        bins, selectdays, ns, max_doy = create_binsamples(oat,'random')
+        # puts("--- bins = #{bins}")
+        # puts("--- selectdays = #{selectdays}")
+        # puts("--- ns = #{ns}")
+        puts("### ============================================================")
+        puts("### Running simulation on samples...")
+        y_seed = run_samples(model, year, selectdays, num_timesteps_in_hr)
+        # puts("--- y_seed = #{y_seed}")
+      elsif load_prediction_method == 'part year bin sample'
+        puts("### Creating bins...")
+        bins, selectdays, ns, max_doy = create_binsamples(oat,'sort')
+        # puts("--- bins = #{bins}")
+        # puts("--- selectdays = #{selectdays}")
+        # puts("--- ns = #{ns}")
+        puts("============================================================")
+        puts("### Running simulation on part year samples...")
+        y_seed = run_part_year_samples(model, max_doy=max_doy, selectdays=selectdays, num_timesteps_in_hr=num_timesteps_in_hr)#, epw_path=epw_path)
+        # puts("--- y_seed = #{y_seed}")
+      end
+      puts("### ============================================================")
+      puts("### Creating annual prediction...")
+      annual_load = load_prediction_from_sample(model, y_seed, bins)
+      # puts("--- annual_load = #{annual_load}")
+      puts("--- annual_load.size = #{annual_load.size}")
+    else
+      puts("### ============================================================")
+      puts("### No load prediction needed...")
+    end
+
+    ############################################
+    # Generate peak schedule
+    ############################################
+    puts("### ============================================================")
+    puts("### Creating peak schedule...")
+    if load_prediction_method == 'fix'
+      puts("### Fixed schedule...")
+      template = 'ComStock 90.1-2019'
+      std = Standard.build(template)
+      climate_zone = std.model_standards_climate_zone(model)
```

Comment:
This is now `OpenstudioStandards::Weather.model_get_climate_zone`, with no need to build a Standard class: https://github.com/NREL/openstudio-standards/blob/8f948207d6af73165a2a8232559804b93d8e50c2/lib/openstudio-standards/weather/information.rb#L92